src/ToDo.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Form, Button, Row, Col } from 'react-bootstrap';
+
+const ToDo = ({todo, index, handleToggle, markTodo, removeTodo}) => {
+    //get what a user click
+    const handleClick = (e) => {
+        e.preventDefault()
+        handleToggle(e.currentTarget.id) //get the index of items
+    }
+
+    //itself. no need to using <toDo></toDo>
+    return (
+        <div>
+        <Row>
+            <Col sm="auto">
+            <Form>
+                <Form.Check onClick={() => markTodo(index)}></Form.Check>{' '}
```

Comment:
Consider self closing the tag when there is nothing in between

i.e.
`<Form.Check onClick={() => markTodo(index)}/>`